independent-publisher-connectors/ORB Intelligence/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+    "properties": {
+        "connectionParameters": {
+            "api_key": {
+                "type": "securestring",
+                "uiDefinition": {
+                    "displayName": "apiKey_auth",
+                    "description": "The apiKey_auth for this api",
+                    "tooltip": "Provide your apiKey_auth",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#da3b01",
+        "capabilities": [],
+        "publisher": "Aaryan Arora, Ankita Singh",
+        "stackOwner": "ORB INTELLIGENCE, INC"

```

Comment:
@aaryan2134 I can not see the above stack Owner name anywhere in the product or company website, It seems the stack Owner is `"Dun & Bradstreet, Inc.`"

Reply:
Made the change